### PR TITLE
feat(utils): ignore fields in metadata

### PR DESCRIPTION
**Pull request by tvaintrob**

### CODE REVIEW

File: src/events/sql.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const uuid4 = require('uuid4');
-const { parse } = require('../resource_utils/sql_utils.js');
+const { parse } = require('../resource_utils/sqs_utils');
```
Comment (ranrib) [inline src/events/sql.js]: Why?

**Comment by tvaintrob** [comment]
had errors on that line trying to run the tests, said it couldn't find the
module that change fixed it

File: src/events/sql.js (hunk 1 above)
Comment (tvaintrob) [reply src/events/sql.js]: had errors on that line trying to run the tests, said it couldn't find the
module that change fixed it

**Review by ranrib — COMMENTED** [review]
Looks good, please add an example to README

**Review by ranrib — APPROVED** [review]
💯

**Pull request merged**